ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -795,8 +795,8 @@ func (r *RayClusterReconciler) reconcilePods(ctx context.Context, instance *rayv
 		if worker.NumOfHosts <= 0 {
 			worker.NumOfHosts = 1
 		}
-		numExpectedPods := int(workerReplicas * worker.NumOfHosts)
-		diff := numExpectedPods - len(runningPods.Items)
+		workerPodsCnt := int(workerReplicas)
```

Comment:
it's a bit weird to see `workerPodsCnt := int(workerReplicas)`.

Maybe we should rename `workerReplicas` to `numExpectedWorkerPods`.

```
numExpectedWorkerPods := int(utils.GetWorkerGroupDesiredReplicas(ctx, worker))
```